test/algorithms/excited_state_solvers/test_bosonic_esc_calculation.py

```diff
@@ -124,6 +124,22 @@ def test_vqe_uvccsd_factory(self):
         for idx, energy in enumerate(self.reference_energies):
             self.assertAlmostEqual(results.computed_vibrational_energies[idx], energy, places=1)
 
+    def test_vqe_uvcc_factory_with_user_initial_point(self):
+        """Test VQEUVCCFactory when using it with a user defined initial point."""
+
+        optimizer = COBYLA(maxiter=5000)
+        quantum_instance = QuantumInstance(
+            backend=qiskit.BasicAer.get_backend("statevector_simulator"),
+            seed_simulator=algorithm_globals.random_seed,
+            seed_transpiler=algorithm_globals.random_seed,
+        )
+        solver = VQEUVCCFactory(quantum_instance, optimizer=optimizer, initial_point=[1, 2, 3])
```

Comment:
How do we know that this ` initial_point=[1, 2, 3]` does anything? The code does not fail but can we test that some behavior is influenced by that such that is actually functioning as expected. Ie. I suspect the test will pass given other values of initial point, and indeed just leaving it as default. What could be done is to set the optimizer to say a single iter and set the initial point to a prior known optimal point. Then with only one iteration if the initial point is indeed having an effect we should already be at the solution.